documentation/asciidoc/topics/attributes/links.adoc

```diff
@@ -7,22 +7,26 @@
 
 :link_tech_preview: https://access.redhat.com/support/offerings/techpreview/[Red Hat Technology Preview Features Support Scope]
 
-:link_os_load_balancer: https://docs.openshift.com/container-platform/4.2/networking/configuring-ingress-cluster-traffic/configuring-ingress-cluster-traffic-load-balancer.html[Configuring ingress cluster traffic using a load balancer]
+:link_os_load_balancer: https://docs.openshift.com/container-platform/4.5/networking/configuring-ingress-cluster-traffic/configuring-ingress-cluster-traffic-load-balancer.html[Configuring ingress cluster traffic using a load balancer]
```

Comment:
@oraNod it seams to 2.0.x not targeted to OCP 4.5

Reply:
downstream we say 2.0.x is supported on OCP 4.4 and 4.5

Reply:
Yes, sure.